packages/fx-core/src/plugins/resource/spfx/plugin.ts

```diff
@@ -284,29 +289,38 @@ export class SPFxPluginImpl {
           "warn",
           util.format(getStrings().plugins.SPFx.createAppCatalogNotice, tenant.value),
           true,
+          "OK",
           Constants.READ_MORE
         );
         const confirm = res?.isOk() ? res.value : undefined;
         switch (confirm) {
           case "OK":
             try {
+              await progressHandler?.next(DeployProgressMessage.CreateSPAppCatalog);
               await SPOClient.createAppCatalog(spoToken);
             } catch (e: any) {
               return err(CreateAppCatalogFailedError(e));
             }
             let retry = 0;
             appCatalogSite = await SPOClient.getAppCatalogSite(spoToken);
             while (appCatalogSite == null && retry < Constants.APP_CATALOG_MAX_TIMES) {
+              ctx.logProvider?.warning(`No tenant app catalog found, retry: ${retry}`);
               await sleep(Constants.APP_CATALOG_REFRESH_TIME);
               appCatalogSite = await SPOClient.getAppCatalogSite(spoToken);
               retry += 1;
             }
             if (appCatalogSite) {
               SPOClient.setBaseUrl(appCatalogSite);
+              ctx.logProvider?.info(
```

Comment:
Will user get confused if the waiting time is too long and no hint for that since info message will not open output channel automatically?

Reply:
actually output channel is opened automatically in spfx deploy. line 212-214